sys/net/gnrc/pktbuf_static/gnrc_pktbuf_static.c

```diff
@@ -447,7 +448,12 @@ static void *_pktbuf_alloc(size_t size)
     }
     else {
         _unused_t *new = (_unused_t *)(((uint8_t *)ptr) + size);
-        if (prev == NULL) { /* ptr was _first_unused */
+
+        if (((((uint8_t *)new) - &_pktbuf[0]) + sizeof(_unused_t)) > GNRC_PKTBUF_SIZE) {
```

Comment:
IMHO that's perfectly clear, because even forgetting all about operator priorities it still wouldn't make sense to me to give `&` priority over `[]`, but fine.